Update the engine version to 1.1.15 (#10661)
ihhub/fheroes2: changelog.txt

version 1.1.15 (12 April 2026)
- fix inconsistency in the Editor's window titles

> - add "Ravenhold" map into the fheroes2 bundle

Reviewer: ```suggestion
- add "Ravenhold" map to the fheroes2 bundle
```

- add "Apocalypse" map into the fheroes2 bundle

> - add "Final Invasion" map into the fheroes2 bundle

Reviewer: Same as above.

- update the Ukrainian translation
- update the Russian translation

> - replace multiple variants of "Eruption" map with a single one with multiple languages

Reviewer: ```suggestion
- consolidate all translated versions of the "Eruption" map into a single map file
```

- add ability to show Turn Order window below the battlefield
- add popup windows for all buttons for the Virtual Keyboard
- rework Battle Only interface
- add ability to select artifacts for Random Artifact objects in the Editor
- fix rendering deadlock for vertical text when a character width is bigger than the maximum width
- fix issues for the text input window

> - replace multiple variants of "7 deserts" map with a single one with multiple languages

Reviewer: ```suggestion
- consolidate all translated versions of the "7 deserts" map into a single map file
```

- add NBSP character to all fonts
- update the Greek translation
- update the Belarusian translation

> - add ABOUT button to show map information during gameplay

Reviewer: ```suggestion
- make the ABOUT button available from the scenario info during gameplay
```

- speed up engine loading and language switching
- recalculate hero path after "Dimension Door" spell usage

> - fix rendering of dead monsters during Resurrection spells animation

Reviewer: ```suggestion
- fix rendering of dead monsters during Resurrection spell animation
```

> - AI heroes will sort artifacts from worst to best to diminish their defeat

Reviewer: ```suggestion
- make AI heroes sort artifacts from worst to best to minimize losses on defeat
```

- separate battle animation Speed string from monster Speed
- add Brazilian Portuguese language support for the Virtual Keyboard
- remove death monster animation in the castle's well dialog
- adjust castle image and castle name in Town's dialog
- add Norwegian language support for the Virtual Keyboard
- add Swedish language support for the Virtual Keyboard
- add Hungarian language support for the Virtual Keyboard
- add Spanish language support to Virtual Keyboard
- add Bulgarian language support for the Virtual Keyboard
- update CP-1253 fonts for the Greek language

> - relax the conditions of action object placement in the Editor

Reviewer: ```suggestion
- Loosen the conditions for action object placement in the Editor
```

> - improve adventure map spells messages order

Reviewer: ```suggestion
- improve adventure map spells message order
```

- move scrollbar only to discrete positions corresponding to the list items
- avoid terrain objects being placed under road objects in the Random Map Generator
- update object passabilities after the Random Map Generator finishes generating map
- compress fheroes2 assets data in h2d file
- add Timidity PAT files to PS Vita package
- generate new sprites for diagonal road crossings
- update the Portuguese Translation
- adjust morale and luck descriptions
- update the Polish translation
- show water percentage limit in Random Generator
- fix vertical offset for small monster images

> - add support to change terrain type in the Battle Only by mouse wheel

Reviewer: ```suggestion
- make it possible to change terrain type in the Battle Only mode with the mouse wheel
```

- fix animation freezing while playing the sound of Petrification spell

> - set 'exchange all troops' default hotkey to 'X', avoid clashing with Mage Guild

Reviewer: ```suggestion
- set 'exchange all troops' default hotkey to 'X' to avoid conflicting with Mage Guild
```
